packages/jobs/package.json

```
@@ -15,30 +15,31 @@
         "directory": "packages/jobs"
     },
     "dependencies": {
+        "@kubernetes/client-node": "^1.3.0",
```

```suggestion
        "@kubernetes/client-node": "1.3.0",
```
we try to pin everything to avoid problems